### PR TITLE
[AWS Ops Works] convert --custom_json and --instance_ids options in a complex object

### PR DESCRIPTION
**Pull request by altherlex**
Resolved comands like that:

``` bash
dpl --provider=opsworks --app-id=***** --wait-until-deployed --instance_ids='102030c************' --instance_ids='["10c8********************************"]' --custom_json='{"deploy": {"espn_api": {"scm": { "revision": "development" }, "environment_variables": {"RAILS_ENV": "staging"} }} }'
```

### CODE REVIEW

**Comment by BanzaiMan** [comment]
What is the motivation for this PR?

**Comment by altherlex** [comment]
Hello Banzai. I need to do something like this:

`dpl --provider=opsworks --app-id=***** --wait-until-deployed --instance_ids='102030c************' --instance_ids='["10c8********************************"]' --custom_json='{"deploy": {"espn_api": {"scm": { "revision": "development" }, "environment_variables": {"RAILS_ENV": "staging"} }} }'`

Because I can't send a Hash (--custom_json) and an Array (--instance_ids) on bash command.

File: lib/dpl/provider/ops_works.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@ def check_auth
       end
 
       def custom_json
+        options[:custom_json] = YAML.load(options[:custom_json]) if options[:custom_json].is_a? String
```
Comment (BanzaiMan) [inline lib/dpl/provider/ops_works.rb]: Are you loading option value named `custom_json` with the YAML loader? That seems incorrect to me.

Comment (BanzaiMan) [inline lib/dpl/provider/ops_works.rb]: Also, why is there an `if` here?

**Comment by BanzaiMan** [comment]
Please add specs.

**Comment by BanzaiMan** [comment]
I'm closing this now, since it's become stale. If anyone is interested in pursuing this further, please open a new PR.

**Pull request closed without merging**